src/locutus_util/helpers.py

```diff
@@ -1,33 +1,31 @@
 import subprocess
 import logging
+import sys
 import time
+import pandas as pd
+from pathlib import Path
 from google.cloud import firestore
 from locutus_util.common import (COL_TIME_LIMIT,SUB_TIME_LIMIT,BATCH_SIZE)
 
 
 def set_logging_config(log_file):
-
-    LOGGING_FORMAT='%(asctime)s - %(levelname)s - %(message)s'
-
-    # Create a logger
     logger = logging.getLogger()
     logger.setLevel(logging.INFO)
-    
-    # Create a file handler for logging to a file
-    file_handler = logging.FileHandler(log_file)
-    file_handler.setLevel(logging.INFO)
-    file_handler.setFormatter(logging.Formatter(LOGGING_FORMAT))
-    
-    # Create a console handler for logging to the console
+
+    # Prevent duplicate handlers
+    if logger.hasHandlers():
+        logger.handlers.clear()
+
+    # Console handler
     console_handler = logging.StreamHandler()
-    console_handler.setLevel(logging.INFO)
-    console_handler.setFormatter(logging.Formatter(LOGGING_FORMAT))
-    
-    # Add handlers to the logger
-    logger.addHandler(file_handler)
+    console_handler.setFormatter(logging.Formatter('%(asctime)s - %(levelname)s - %(message)s'))
     logger.addHandler(console_handler)
 
-    logger.info('logger is setup')
+    # File handler
+    file_handler = logging.FileHandler(log_file)
+    file_handler.setFormatter(logging.Formatter('%(asctime)s - %(levelname)s - %(message)s'))
+    logger.addHandler(file_handler)
+
 
```

Comment:
Logger was printing twice in the terminal. Not helpful when looking at logs as a sanity check.